### ISSUE TITLE
Missing sorting option for severity of the issues / bugs

### ISSUE THREAD

**Issue by Lynaga** [corynz/bugwalker-web#22]
There are no sorting for severity.

![image](https://user-images.githubusercontent.com/5549222/94996361-9d7e2f00-05a4-11eb-911f-ab8a19ef4127.png)

**Comment by corynz** [comment]
You're right. It's possible to filter by severity but not to sort by it, marked as enhancement
